doc/support_matrix.yml

```
@@ -494,6 +494,15 @@ boards:
       user_usb: not_currently_supported
       ethernet_over_usb: not_currently_supported
 
+  heltec-v3:
```

This build/context name will need to be updated in the rest of the code:

```suggestion
  heltec-wifi-lora-32-v3:
```